zfs-prune-snapshots

```diff
@@ -53,23 +53,28 @@ timespec
 	y years
 
 options
-    -h             print this message and exit
-    -n             dry-run, don't actually delete snapshots
-    -l             list only mode, just list matching snapshots names
-		   without deleting (like dry-run mode with machine-parseable
-		   output)
-    -p <prefix>    snapshot prefix string to match
-    -s <suffix>    snapshot suffix string to match
-    -i             invert matching of prefix and suffix
-    -q             quiet, do not printout removed snapshots
-    -R             recursively delete, pass '-R' directly to 'zfs destroy'
-    -v             increase verbosity
-    -V             print the version number and exit
+    -h             Print this message and exit
```

Comment:
I see that you've added lines for `-d` and `-r`, etc. but you've also changed the beginning of each line to uppercase.  I'm happy to have a discussion on a PR for changing the casing, but as it stands right now it's half uppercase and half lowercase.... you can see this when running with `-h`.

I'd prefer you just leave the casing as-is and if you'd like to make changes to it we can handle it in a separate PR.

Personally, I have no strong feelings either way when it comes to uppercase vs lowercase in code... i tend to just not use capitals when i type out of laziness ha

Basically, i'd just prefer to keep style changes separate from function changes.

Reply:
sure, I just pushed it quickly, I will revert to lower casing